ui/src/app/modules/status/widgets/update-info-widget/update-info-widget.component.ts

```diff
@@ -48,24 +55,36 @@ export class UpdateInfoWidgetComponent implements OnInit {
   public packageVersion = this.$settings.env.packageVersion
   public homebridgeVersion = this.$settings.env.homebridgeVersion
 
+  public dockerInfo: DockerDetails = {
+    currentVersion: undefined,
+    latestVersion: null,
+    latestReleaseBody: '',
+    updateAvailable: false,
+  }
+
+  public dockerStatusDone = false as boolean
+  public dockerExpanded = false
+
   public async ngOnInit() {
     this.io = this.$ws.getExistingNamespace('status')
 
     this.io.connected.subscribe(async () => {
+      await this.getNodeInfo()
```

Comment:
i was trying to see why you might have moved this out of the Promise.all ? (and below)

Reply:
This way nodeInfo is populated before the other functions are called.